xrpl/asyncio/transaction/main.py

```diff
@@ -142,6 +147,76 @@ async def submit_transaction(
     raise XRPLRequestFailureException(response.result)
 
 
+async def safe_sign_transaction_with_multisign(
+    transaction: Transaction,
+    wallet: Wallet,
+) -> str:
+    """
+    Signs a transaction with multisign locally, without trusting external rippled nodes.
+
+    Args:
+        transaction: the transaction to be signed.
+        wallet: the wallet with which to sign the transaction.
+
+    Returns:
+        The signed transaction blob.
+    """
+    signature = sign(
+        bytes.fromhex(
+            encode_for_multisigning(
+                transaction.to_xrpl(),
+                wallet.classic_address,
+            )
+        ),
+        wallet.private_key,
+    )
+
+    tx_dict = transaction.to_dict()
+    tx_dict["signers"] = [
+        Signer(
+            account=wallet.classic_address,
+            txn_signature=signature,
+            signing_pub_key=wallet.public_key,
+        )
+    ]
+
+    return encode(Transaction.from_dict(tx_dict).to_xrpl())
+
+
+async def multisign(transaction: Transaction, tx_blobs: List[str]) -> Transaction:
+    """
+    Takes several transactions with Signer fields (blob form) and creates a
+    single transaction with all Signers that then gets signed and returned.
+
+    Args:
+        transaction: the transaction to be multisigned.
+        tx_blobs: a list of signed transactions (in blob form) to combine into
+            a single signed transaction.
+
+    Returns:
+        The multisigned transaction.
+    """
+    # Individually signing the transaction with each signer.
+    # This would normally be handled by reaching out to the signing
+    # account owners in your application since you would not normally
+    # be in control of all the keys.
```

Comment:
We don't need this part of the comment here since this is a utility method instead of a snippet doing something a bit unorthodox.